src/reflect/scala/reflect/internal/StdNames.scala

```diff
@@ -351,7 +358,7 @@ trait StdNames {
     val MIRROR_UNTYPED: NameType           = "$m$untyped"
     val REIFY_FREE_PREFIX: NameType        = "free$"
     val REIFY_FREE_THIS_SUFFIX: NameType   = "$this"
```

Comment:
was there no warning here?

Reply:
Sharp eyes! I'll wonder if this isPrivateLocal when I'm fully awake.

Reply:
I need a couple more days of sleep after the national election, then I'll wonder this. Or `"$this"`.